pipeline_1_filter.sh

```diff
@@ -1,42 +1,15 @@
 #!/bin/bash
-GATK_INPUT="${VEP_OUTPUT}"
-GATK_INPUT_INDEX="${GATK_INPUT}".tbi
-GATK_OUTPUT_DIR="${OUTPUT_DIR}"/step1_filter
-GATK_OUTPUT="${GATK_OUTPUT_DIR}"/"${OUTPUT_FILE}"
+FILTER_INPUT="${VCFANNO_OUTPUT}"
+FILTER_OUTPUT_DIR="${OUTPUT_DIR}"/step1_filter
+FILTER_OUTPUT="${FILTER_OUTPUT_DIR}"/"${OUTPUT_FILE}"
 
-mkdir -p "${GATK_OUTPUT_DIR}"
-
-if [ -f "${GATK_INPUT_INDEX}" ]
-then
-        if [ "$FORCE" == "1" ]
-        then
-                rm "${GATK_INPUT_INDEX}"
-        else
-                echo "${GATK_INPUT_INDEX} already exists, use -f to overwrite.
-                "
-                exit 2
-        fi
-fi
-if [ -f "${GATK_OUTPUT}" ]
-then
-        if [ "$FORCE" == "1" ]
-        then
-                rm "${GATK_OUTPUT}"
-        else
-                echo "${GATK_OUTPUT} already exists, use -f to overwrite.
-                "
-                exit 2
-        fi
-fi
+mkdir -p "${FILTER_OUTPUT_DIR}"
 
+module load BCFtools
 module load HTSlib
-tabix "${GATK_INPUT}"
-module unload HTSlib
 
-module load GATK
-gatk VariantFiltration \
-   -V "${GATK_INPUT}" \
-   -O "${GATK_OUTPUT}" \
-   --filter-expression "DP > 25" \
-   --filter-name "DP"
-module unload GATK
+bcftools filter -i'CAP[*]>0.9' "${FILTER_INPUT}" | \
```

Comment:
this discards records with missing CAP field or records with a CAP field with missing scores. what should we do with these cases?